### ISSUE TITLE
Unable to import ONNX file from PyTorch pre-trained BERT model

### ISSUE THREAD

**Issue by mvermeulen** [ROCmSoftwarePlatform/AMDMIGraphX#266]
Tried generating an ONNX file from a PyTorch pre-trained BERT instance found at the [following url](https://modelzoo.co/model/pytorch-pretrained-bert)

The script used to create the ONNX file is attached (remove the .txt to get a .py file).  This was created from the example at the model-zoo above.
[export_bert.py.txt](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/files/3229724/export_bert.py.txt)

When running this under read_onnx, I see the following failure
```
mev@chilecito:~/source/AMDMIGraphX/build/src/onnx$ ./read_onnx /home/mev/bert/bertmodel.onnx 
[libprotobuf INFO google/protobuf/io/coded_stream.cc:610] Reading dangerously large protocol message.  If the message turns out to be larger than 2147483647 bytes, parsing will be halted for security reasons.  To increase the limit (or to disable these warnings), see CodedInputStream::SetTotalBytesLimit() in google/protobuf/io/coded_stream.h.
[libprotobuf WARNING google/protobuf/io/coded_stream.cc:81] The total number of bytes read was 438008057
terminate called after throwing an instance of 'migraphx::version_1::exception'
  what():  /home/mev/source/AMDMIGraphX/src/include/migraphx/check_shapes.hpp:81: Types do not match
Aborted (core dumped)
```

**Comment by kahmed10** [comment]
These are the operators I found in bertmodel.onnx:
    op_type: "Add"
    **op_type: "Cast"**
    op_type: "Concat"
    op_type: "Constant"
    **op_type: "ConstantOfShape"**
    op_type: "Div"
    **op_type: "Erf"**
    **op_type: "Expand"**
    op_type: "Gather"
    op_type: "Gemm"
    op_type: "MatMul"
    op_type: "Mul"
    **op_type: "Pow"**
    op_type: "ReduceMean"
    op_type: "Reshape"
    op_type: "Shape"
    op_type: "Softmax"
    op_type: "Sqrt"
    op_type: "Sub"
    op_type: "Tanh"
    op_type: "Transpose"
    op_type: "Unsqueeze"

bermodel_lm.onnx is a subset of the operators above and contains all of the missing operators.

**Comment by mvermeulen** [comment]
An updated create_bert.py script is attached.
[create_bert.py.txt](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/files/3326931/create_bert.py.txt)

This one has some options for batch sizes (--batch_size), different bert models (--bert_model) and different sequence lengths (--max_seq_length).  It also creates both the original fully trained BERT and pre-trained variants that require additional fine tuning.  This ONNX file has the same operators as the list above.  I have also verified that the derivative models do not introduce additional operators.

**Comment by mvermeulen** [comment]
Fixed
